GithubPractice/ViewController.swift

```diff
@@ -7,19 +7,25 @@
 
 import UIKit
 
-class ViewController: UIViewController {
+class ViewController: UIViewController,UITextFieldDelegate {
```

Comment:
select all code and format it with control+I

Reply:
@elanghel we typically prefer to have spacing between all instructions:
`let sum = 3+5` -> `let sum = 3 + 5`
`guard name="John" else { ... }` -> `guard name = "John" else { ... }`

Reply:
Also, please check this:
https://github.com/airbnb/swift

Reply:
> Also, please check this:
> https://github.com/airbnb/swift

@Vasile95, okay thanks